.werft/values.k3sWsDisable.yaml

```diff
@@ -3,9 +3,6 @@ components:
   proxy:
```

Comment:
This is actually not a rename but a file dedicated to disable meta component installation on ws cluster

Reply:
makes sense. Maybe name it "disableMeta" or (if it is used only together with `values.k3sWsCluster.yaml`) embed the values into `values.k3sWsCluster.yaml`?

Reply:
We set and generate some flags dynamically. These flags are appended to the `commonFlags` and `flags` variable. To avoid them getting overridden I have split the disable logic into a dedicated file which is prioritized over other files. I feel that this keeps it clean.

I will rename this to `disableMeta`, that is more intuitive.